11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,75 @@
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    /*font-size: 100%;
+    font: inherit;*/
+    vertical-align: baseline;
+}
+
+header{
+  background-image:url("../img/bg.jpg");
+  height:700px;
```

Comment:
Maybe play with background-position here to make the view seem more dynamic. `background-position:center;` gives a nice feel.

See more: http://www.w3schools.com/css/tryit.asp?filename=trycss_background-image_position